manual/index.html

```diff
@@ -492,17 +490,21 @@ <h1><a href="https://threejs.org">three.js</a></h1>
 				// We can have 2 hashes. One for the main page, one for the page it's referencing
 				// In other words
 				//	 #en/somePage#someSectionOfPage
-				const subHash = splitHash[0].indexOf('#');
+				const subHash = splitHash[ 0 ].indexOf( '#' );
 				let src;
-				if (subHash >= 0) {
-					const beforeSubHash = splitHash[0].slice(0, subHash);
-					const afterSubHash = splitHash[0].slice(subHash);
+				if ( subHash >= 0 ) {
+
+					const beforeSubHash = splitHash[ 0 ].slice( 0, subHash );
+					const afterSubHash = splitHash[ 0 ].slice( subHash );
 					src = `${beforeSubHash}.html${afterSubHash}${splitHash[ 1 ]}`;
+		
 				} else {
+
 					src = splitHash[ 0 ] + '.html' + splitHash[ 1 ];
+		
 				}
 
-				iframe.src = src;  // lgtm[js/client-side-unvalidated-url-redirection]
+				iframe.src = src; // lgtm[js/client-side-unvalidated-url-redirection]
```

Comment:
Maybe we should remove this comment? We now use CodeQL instead of LGTM.

Reply:
Let's do this in a separate PR after this one has been merged since there is another annotation in `manual/resources/lesson.js`

Reply:
FYI: Removing the annotations triggers CodeQL warnings.

Reply:
Maybe then we can do a PR where we can add something similar (maybe there is a new CodeQL-special pattern) for all places where CodeQL triggers incorrectly.